using-valtimo/document/custom-case-columns.md

```diff
@@ -0,0 +1,162 @@
+# Custom case columns
+
+Each case comes with a default case overview with four standard columns. 'Reference number', 'Created by', 'Created on',
+and 'Last modified'. These columns can be adjusted. Any key defined in a document can be used as a column in a case
```

Comment:
The use of the word adjusted sounds like these specific columns can be tweaked a bit. I would change the sentence to something like this: 
```
The columns in the list can be changed, added and removed.
```